source/client/process_impl.cc

```diff
@@ -244,6 +244,11 @@ void ProcessImpl::createBootstrapConfiguration(envoy::config::bootstrap::v2::Boo
       }
     }
 
+    if (options_.transportSocket().has_value()) {
+      auto* transport_socket = cluster->mutable_transport_socket();
```

Comment:
This may not be a good use of auto, since it hides useful type information from the reader. Can we expand it into the type name?

Reply:
Simplified to a single line now.